src/components/Header/Searchbar/Searchbar.component.jsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import styled from 'styled-components';
+import {motion} from 'framer-motion';
+
+const Container = styled(motion.div)`
+    align-items: center;
+    background-color: transparent;
+    display: flex;
+    width: 100%;
+`;
+
+const InnerContainer = styled(motion.div)`
+    align-items: center;
+    display: flex;
+    margin: auto;
+    width: 50%;
+`;
+
+const StyledSearchbar = styled(motion.input)`
```

Comment:
This looks good, you can improve this by moving the styled components to their own files, this improves maintainability and readability